added 67-PA-T-129A.md

zdover23 opened the pull request:
I have added a short memo that had not yet been included in the repository, as a test of the workflow. If this works, I'll do more of these.

seanredmond commented:
Thanks! There are a couple of practices I've been following that I really need to document for contributors. Can you update this PR with these changes:

1. Add line breaks to the markdown where they fall in the original (they don't show up in the web page, but at least they're recorded in the source). When a word is hyphenated, just "dehyphenate" it and put a newline after it.
2. Don't transcribe the signature or addressees

Thanks again.

zdover23 commented:
Sean,

I have updated this PR, adding line breaks in the .md files where the line breaks appear in Tindall's original memos. I have removed the signatures, including the transcription of Tindall's name under the signature.

If you agree to another PR, I will write a procedure explaining to people who want to contribute how to do so.

seanredmond commented:
Fixed up a few typos and merged. Thanks
